WebApp/autoreduce_webapp/instrument/views/helper.py

```diff
@@ -0,0 +1,21 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+from queue_processors.queue_processor.status_utils import STATUS
+
+
+def get_last(runs_for_instrument):
```

Comment:
Since django is active record pattern, wouldn't this make more sense as either a method on  the `ReductionRun` model that takes an instrument as arg or a method on the `Instrument` model. Plus we dont end up with even more utils and helpers

Reply:
Yes that makes more sense. I thought about it but decided not to as it was on the FK relationship, and forgot I can just put it in the Instrument model!